### PR TITLE
Bump parent pom from 4.58 to 4.63

### PR DESCRIPTION
**Pull request by ampuscas**
Bump parent pom so that we have the latest version of `jenkins-test-harness` which will fix several test failures

### Submitter checklist

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -150,6 +150,7 @@ THE SOFTWARE.
             <groupId>org.mockito</groupId>
             <artifactId>mockito-inline</artifactId>
             <scope>test</scope>
+            <version>5.2.0</version>
```
Comment (jtnord) [inline pom.xml:153]: this is not correct - `mockito-inline` is no more and should be just replaced with `mockito` 

When bumping parent always be sure to heck all the release notes :)
https://github.com/jenkinsci/plugin-pom/releases/tag/plugin-4.60

File: pom.xml [hunk 2]
```diff
@@ -30,7 +30,7 @@ THE SOFTWARE.
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.58</version>
+        <version>4.63</version>
```
Comment (jtnord) [inline pom.xml:33]: `4.64` is available - any reason this is not to be used?

Comment (ampuscas) [reply pom.xml]: It wasn't available when I created this PR :) - 4 days ago. I updated it now

**Review by jtnord — APPROVED** [review]

**Comment by car-roll** [comment]
@ampuscas I believe this PR is no longer needed

**Comment by ampuscas** [comment]
Yes, I am closing it

**Pull request closed without merging**